### PR TITLE
Navigation bar added

### PR DESCRIPTION
**Pull request by Reena-Koudi**

### CODE REVIEW

**Review by Hemesh-Unka — CHANGES_REQUESTED** [review]
Hi Reena,

Are we able to get the users name and sign out option in the navbar when logged in?

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -17,6 +20,7 @@
       <p class="alert alert-danger"><%= alert %></p>
     <% end %>
 
+    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRpYw8zT74fI_hEC1oWuxBffkeucCoirAe-G_hjrjyNmIXJCCX4TQ" class="rounded float-right">
```
Comment (Hemesh-Unka) [inline app/views/layouts/application.html.erb:23]: Not sure what this is for?

Comment (Hemesh-Unka) [inline app/views/layouts/application.html.erb:23]: This code needs to be moved up into the navbar

Comment (chiakimz) [reply app/views/layouts/application.html.erb:23]: Needs some adjustment but looking cool!

**Review by chiakimz — APPROVED** [review]
It looks pretty, I like how pink it is.

**Review by Hemesh-Unka — APPROVED** [review]

**Pull request merged**